test/templates/message-broker-with-entities-2nd-call/src/main/resources/config/application.yml

```diff
@@ -161,12 +161,6 @@ jhipster:
 
 kafka:
   bootstrap.servers: ${KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
-  producer:
-    foo:
-      name: queuing.message_broker_with_entities.foo
-      enabled: true
-      '[key.serializer]': org.apache.kafka.common.serialization.StringSerializer
-      '[value.serializer]': com.mycompany.myapp.service.kafka.serializer.FooSerializer
```

Comment:
I think @monxxi put this `producer:` block here to ensure generation order blocks. There is a test especially for it.

Reply:
So it should stay here for the tests, that's why it's not always a good idea to run a `yo jhipster-kafka` in templates folders in my opinion.